content/docs/querying/operators.md

```diff
@@ -195,7 +196,7 @@ If the metric `http_requests_total` had time series that fan out by
 `application`, `instance`, and `group` labels, we could calculate the total
 number of seen HTTP requests per application and group over all instances via:
 
-    sum(http_requests_total) by (application, group)
+    sum(http_requests_total) without (instance)
```

Comment:
For the vast majority of usage I think `without` is preferable in practice. The sort of labels you're not listing are the ones you usually don't care about in rules as they're not being matched on anything (they're more useful up in the AM and at high level aggregations).